src/common/methods.ts

```
@@ -37,7 +37,62 @@ export const ordinalSuffixOf = (n: number) => {
+    return getImgurImageSized(imageUrl, size)
+  }
+
+  if (/digitaloceanspaces\.com/.test(imageUrl)) {
+    return getS3ImageSized(imageUrl, resolvedSize)
+  }
+
+  // Fallback based on declared or defaulted source
+  switch (imageSource) {
+    case 'aws':
+      return getS3ImageSized(imageUrl, resolvedSize)
+    case 'imgur':
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)